owncloudApp/src/main/java/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -468,21 +468,15 @@ public void onCreate(Bundle savedInstanceState) {
             }
         }
 
-        boolean loggerEnabled = getResources().getBoolean(R.bool.logger_enabled) ||
-                BuildConfig.DEBUG || MainApp.isBeta();
-        Preference pLogger = findPreference("logger");
-        if (pLogger != null) {
-            if (loggerEnabled) {
-                pLogger.setOnPreferenceClickListener(preference -> {
-                    Intent loggerIntent = new Intent(getApplicationContext(), LogHistoryActivity.class);
-                    startActivity(loggerIntent);
+        // show item(s) only when you are developer
+        mLogger = findPreference("logger");
```

Comment:
I know the rest of the preferences are doing the same but "logger" should be in a constant and not hardcoded

Reply:
feel free to do it